exercises/simple-cipher/description.md

```diff
@@ -11,7 +11,7 @@ for A, and so with the others."
 
 Ciphers are very straight-forward algorithms that allow us to render
 text less readable while still allowing easy deciphering. They are
-vulnerable to many forms of cryptoanalysis, but we are lucky that
+vulnerable to many forms of cryptanalysis, but we are lucky that
 generally our little sisters are not cryptoanalysts.
```

Comment:
so! should cryptoanalysts change to cryptanalysts to mirror the above change?

Reply:
Thanks, yes, [usage](https://books.google.com/ngrams/graph?content=cryptoanalyst%2Ccryptanalyst) suggests so. Apparently it's incorrect in my spell checker's dictionary.